src/spikeinterface/widgets/all_amplitudes_distributions.py

```diff
@@ -9,16 +9,22 @@
 
 class AllAmplitudesDistributionsWidget(BaseWidget):
     """
-    Plots distributions of amplitudes as violon plot for all or some units.
+    Plots distributions of amplitudes as violin plot for all or some units.
 
     Parameters
     ----------
     waveform_extractor: WaveformExtractor
         The input waveform extractor
     unit_ids: list
-        List of unit ids.
+        List of unit ids, default None
     unit_colors: None or dict
-        Dict of colors
+        Dict of colors with key: unit, value: color, default None
+    backend : None or str
+        Three possible options:
+        * 'matplotlib': uses matplotlib backend
+        * 'ipywidgets': can only be used in Jupyter notebooks/Jupyter lab
+        * 'sortingview': for web-based GUIs
+        Default is None which uses the matplotlib backend
```

Comment:
We have a system to inject automatically more doc that is backend specific here
https://github.com/SpikeInterface/spikeinterface/blob/main/src/spikeinterface/widgets/widget_list.py#L78
Maybe we should do this there to avoid copy/paste.

Reply:
You're right. I'll fix that tomorrow.